cms/templates/js/publish-xblock.underscore

```diff
@@ -101,6 +101,10 @@ var visibleToStaffOnly = visibilityState === 'staff_only';
             </a>
             </li>
         </ul>
+        <p>
+            <%- gettext("Note: Do not hide graded assignments using this feature.") %>
```

Comment:
Is "this feature" clear to the user?  What feature, in particular, are we talking about here?